src/server/modules/request/request.service.ts

```diff
@@ -111,7 +114,7 @@ function parsePipeline(pipeline: any): any[] {
   try {
     return parse('[' + objectToString(pipeline[0]) + ']');
   } catch (e) {
-    console.error('Parsing error on pipeline');
+    console.debug('Parsing error on pipeline');
```

Comment:
What is the reason for changing the log level of the error ?

Reply:
I forgot this modification. 

In fact it is no longer necessary, this method will soon disappear.
I will revert it thank you.

I just wanted to stop spamming logs when Maggregor tried to parse pipelines that wasn't pipeline. But actually it's a subject much better covered in this PR: https://github.com/maggregor/maggregor/pull/30